sources/packages/web/src/components/institutions/active-application/ActiveApplicationSummaryData.vue

```diff
@@ -1,85 +1,90 @@
 <template>
-  <body-header
-    title="Applications"
-    :recordsCount="applications.results?.length"
-    class="m-1"
+  <full-page-container
+    :layout-template="LayoutTemplates.CenteredCard"
```

Comment:
CenteredCard is the default property value of layout-template. So not required to explicitly assign.